src/sgame/sg_weapon.cpp

```diff
@@ -636,7 +625,7 @@ HIVE
 Target tracking for the hive missile.
 ================
 */
-static void HiveMissileThink( gentity_t *self )
+void HiveMissileThink( gentity_t *self )
```

Comment:
nit: Since this is a global function, should we do the G_ prefix? Should this function just be moved to the Component? (same comment for the rocket pod)

Reply:
I am working on CBSE for missiles, so when the hive missile has its own component I can move it there

Reply:
Sgtm